com.unity.render-pipelines.high-definition/Runtime/ShaderLibrary/ShaderVariables.hlsl

```diff
@@ -235,6 +235,11 @@ float SampleCameraDepth(float2 uv)
     return LoadCameraDepth(uint2(uv * _ScreenSize.xy));
 }
 
+float4 GatherCameraDepth(float2 uv)
+{
+    return GATHER_RED_TEXTURE2D_X(_CameraDepthTexture, s_trilinear_clamp_sampler, uv * _RTHandleScale.xy);
```

Comment:
If we keep this, the above still applies, we shouldn't trilinear here.

Reply:
Removing this, unused